tools/ld-chroma-decoder/palcolour.cpp

```diff
@@ -519,7 +551,10 @@ void PalColour::decodeLine(const SourceField &inputField, const ChromaSample *ch
     // burst-based correction applied.
     const double scaledSaturation = 2.0 * scaledContrast * chromaGain;
 
-    for (qint32 i = videoParameters.activeVideoStart; i < videoParameters.activeVideoEnd; i++) {
+    // extract luma first so it can be run through NR
+    double extractedY[videoParameters.fieldWidth];
```

Comment:
I don't think it's worth changing this right now, since this loop doesn't vectorise very well at the moment anyway, but it might be worth putting this in an aligned array that's not on the stack in the future.